src/Nethermind/Nethermind.Db/StandardDbInitializer.cs

```diff
@@ -65,6 +62,10 @@ private void RegisterAll(bool useReceiptsDb)
                 RegisterCustomDb(DbNames.Receipts, () => new ReadOnlyColumnsDb<ReceiptsColumns>(new MemColumnsDb<ReceiptsColumns>(), false));
             }
             RegisterDb(BuildRocksDbSettings(DbNames.Metadata, () => Metrics.MetadataDbReads++, () => Metrics.MetadataDbWrites++));
+            if (useBlobDb)
+            {
+                RegisterDb(BuildRocksDbSettings(DbNames.BlobTransactions, () => Metrics.BlobTransactionsDbReads++, () => Metrics.BlobTransactionsDbWrites++));
```

Comment:
You can ask @asdacap for the best RocksDb settings for this new DB

Reply:
We migrated to ColumnsDb. Thanks @asdacap for valuable tips